<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1031</width>
    <height>847</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <property name="leftMargin">
    <number>100</number>
   </property>
   <property name="topMargin">
    <number>12</number>
   </property>
   <property name="rightMargin">
    <number>100</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="8" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QScrollArea" name="scrollArea_2">
       <property name="styleSheet">
        <string notr="true">QScrollArea{
background: #957F5F;
border: none;
}</string>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>412</width>
          <height>83</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget#scrollAreaWidgetContents{
background:rgba(149,127,95,1);

}</string>
        </property>
        <layout class="QVBoxLayout" name="verLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>16</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 16pt &quot;Segoe UI&quot;;
    color: white;</string>
           </property>
           <property name="text">
            <string>El usuario puede VER las columnas:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxAllVer">
           <property name="styleSheet">
            <string notr="true">font: 14pt &quot;Segoe UI&quot;;
color: white;</string>
           </property>
           <property name="text">
            <string>Seleccionar todo</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QScrollArea" name="scrollArea_3">
       <property name="styleSheet">
        <string notr="true">QScrollArea {
background:#957F5F;
border: none;
}</string>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>423</width>
          <height>83</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget#scrollAreaWidgetContents_2{
background:rgba(149,127,95,1);

}</string>
        </property>
        <layout class="QVBoxLayout" name="escribirLayout">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>16</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 16pt &quot;Segoe UI&quot;;
    color: white;</string>
           </property>
           <property name="text">
            <string>El usuario puede ESCRIBIR en las columnas:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxAllEscribir">
           <property name="styleSheet">
            <string notr="true">font: 14pt &quot;Segoe UI&quot;;
color: white;</string>
           </property>
           <property name="text">
            <string>Seleccionar todo</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>100</number>
     </property>
     <property name="rightMargin">
      <number>100</number>
     </property>
     <item row="7" column="0">
      <widget class="QLabel" name="label_6">
       <property name="styleSheet">
        <string notr="true">font: 18pt &quot;Segoe UI&quot;;
color: #666666;</string>
       </property>
       <property name="text">
        <string>Rol</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_error">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Arial&quot;;
color:#de574e;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>18</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 18pt &quot;Segoe UI&quot;;
color: #666666;</string>
       </property>
       <property name="text">
        <string>Usuario</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>18</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 18pt &quot;Segoe UI&quot;;
color: #666666;</string>
       </property>
       <property name="text">
        <string>Contraseña</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLineEdit" name="lineEdit_nombreusuario">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Segoe UI&quot;;
color: #666666;
background-color: white;
border-radius: 8px;
border: 1px solid #CCCCCC;
padding: 5px 10px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Usuario...</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QComboBox" name="comboBox_roles">
       <property name="styleSheet">
        <string notr="true">/* style for the QComboBox */
QComboBox {
font-size: 14pt;
border-radius: 11px;
border: 1px solid #ccc;
background-color: white;
color: #666;
height: 25px;
padding: 0 15px;

}

/* style for drop down area */
QComboBox::drop-down{
border: 0px;
}
/* style for drop down arrow */
QComboBox::down-arrow {
image: url(:/resources/resources/icons/down-arrow-gray.png);
width: 20px;
height: 20px;
margin-right: 15px;
}
/* style for QComboBox after open selcet menu */
QComboBox:on {
border: 1px solid #c2dbfe;
}
/* style for list menu */
QComboBox QListView {
font-size: 16pt;
border: 1px solid rgba(0, 0, 0, 10%);
padding: 5px;
background-color: #fff;
outline: 0px;
}

/* style for list items */
QComboBox QListView::item {
padding-left: 10px;
background-color: #fff;
}
QComboBox QListView::item:hover {
background-color: #d3c393;
}
QComboBox QListView::item:selected {
background-color: #d3c393;
}</string>
       </property>
       <item>
        <property name="text">
         <string>Selecciona un rol...</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Armadores</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Proyectista</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Trámites</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Presupuesto</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Juridico</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Administrador</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Otro</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="mensaje">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Arial&quot;;
color: #8fc26b;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLineEdit" name="lineEdit_contrasenausuario">
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Segoe UI&quot;;
color: #666666;
background-color: white;
border-radius: 8px;
border: 1px solid #CCCCCC;
padding: 5px 10px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Contraseña...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="titulo">
       <property name="styleSheet">
        <string notr="true">font: 30pt &quot;Arial Rounded MT Bold&quot;;
color: #957F5F;
</string>
       </property>
       <property name="text">
        <string>Registrar Usuario</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="9" column="0">
    <layout class="QGridLayout" name="gridLayout_3">
     <property name="leftMargin">
      <number>100</number>
     </property>
     <property name="rightMargin">
      <number>100</number>
     </property>
     <property name="bottomMargin">
      <number>12</number>
     </property>
     <item row="0" column="1">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="button_guardar">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
border-radius: 8px;
background-color: #957F5F;
color: white;
font: 16pt &quot;Segoe UI&quot;;
padding: 10px 30px;
}

QPushButton:hover {
	background-color: rgb(116, 91, 47);
	color: rgb(255, 255, 255);
}

QPushButton:pressed {
    border-style: inset;
    background-color: rgb(103, 80, 41);
	color: rgb(255, 255, 255);
}</string>
       </property>
       <property name="text">
        <string>Confirmar</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_cancelar">
       <property name="styleSheet">
        <string notr="true">QPushButton {
border-radius: 8px;
border: 1px solid #957F5F;
background-color: white;
color: #957F5F;
font: 16pt &quot;Segoe UI&quot;;
padding: 10px 30px;
}

QPushButton:hover {
	background-color: rgb(116, 91, 47);
	color: rgb(255, 255, 255);
}

QPushButton:pressed {
    border-style: inset;
    background-color: rgb(103, 80, 41);
	color: rgb(255, 255, 255);
}</string>
       </property>
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0" rowspan="2">
    <layout class="QGridLayout" name="gridLayout_7">
     <property name="leftMargin">
      <number>100</number>
     </property>
     <property name="rightMargin">
      <number>100</number>
     </property>
     <item row="2" column="0">
      <widget class="QLabel" name="label_tabla_3">
       <property name="styleSheet">
        <string notr="true">font: 16pt &quot;Segoe UI&quot;;
color: #666666;</string>
       </property>
       <property name="text">
        <string>Selecciona una tabla</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QComboBox" name="tablaslist">
       <property name="styleSheet">
        <string notr="true">/* style for the QComboBox */
QComboBox {
font-size: 16pt;
border-radius: 11px;
border: 1px solid #ccc;
background-color: white;
color: #666;
height: 25px;
padding: 0 15px;

}

/* style for drop down area */
QComboBox::drop-down{
border: 0px;
}
/* style for drop down arrow */
QComboBox::down-arrow {
image: url(:/resources/resources/icons/down-arrow-gray.png);
width: 20px;
height: 20px;
margin-right: 15px;
}
/* style for QComboBox after open selcet menu */
QComboBox:on {
border: 1px solid #c2dbfe;
}
/* style for list menu */
QComboBox QListView {
font-size: 12px;
border: 1px solid rgba(0, 0, 0, 10%);
padding: 5px;
background-color: #fff;
outline: 0px;
}

/* style for list items */
QComboBox QListView::item {
padding-left: 10px;
background-color: #fff;
}
QComboBox QListView::item:hover {
background-color: #d3c393;
}
QComboBox QListView::item:selected {
background-color: #d3c393;
}</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">font: 18pt &quot;Segoe UI&quot;;
color: #957F5F;</string>
       </property>
       <property name="text">
        <string>Agregar permisos</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
